app/components/publications/publication/documents/documents-upload-modal.js

```diff
@@ -0,0 +1,86 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency-decorators';
+import { all } from 'ember-concurrency';
+import moment from 'moment';
+
+export default class PublicationsPublicationDocumentsDocumentsUploadModalComponent extends Component {
+  @inject store;
+
+  @tracked isExpanded = false;
+  @tracked newPieces = [];
+
+  @action
+  toggleSize() {
+    this.isExpanded = !this.isExpanded;
+  }
+
+  @action
+  uploadPiece(file) {
+    const now = moment().utc()
```

Comment:
werken met een `moment`-date object is hier overbodig. Kan je deze omvormen naar een natve JS object (`new Date()`)?